### PR TITLE
feat: serverless custom domains docs

### PR DESCRIPTION
**Pull request by johnharris85**
### Description

Add specific serverless docs for custom domains. 

### Testing instructions

Preview link: https://deploy-preview-8213--kongdocs.netlify.app/

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/_data/docs_nav_konnect.yml [hunk 1]
```diff
@@ -62,6 +62,8 @@
           url: /gateway-manager/serverless-gateways/provision-serverless-gateway/
         - text: Securing Backend Traffic
           url: /gateway-manager/serverless-gateways/securing-backend-traffic/
+        - text: Custom Domains
+          url: /gateway-manager/serverless-gateways/custom-dns/
```
Comment (lena-larionova) [inline app/_data/docs_nav_konnect.yml:66]: Please also remove the original nav entry for custom domains from the reference section, and add ones for Dev Portal and Dedicated Cloud Gateways.

File: app/konnect/dev-portal/custom-dns.md [hunk 2]
```diff
@@ -2,32 +2,7 @@
 title: Custom Domains
```
Comment (lena-larionova) [inline app/konnect/dev-portal/custom-dns.md:2]:
```suggestion
title: Custom Domains for Dev Portal
```
For better SEO + internal search purposes.

File: app/konnect/gateway-manager/dedicated-cloud-gateways/custom-dns.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+title: Custom Domains
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/dedicated-cloud-gateways/custom-dns.md:2]:
```suggestion
title: Custom Domains for Dedicated Cloud Gateways
```

File: app/konnect/gateway-manager/dedicated-cloud-gateways/custom-dns.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+  {:.note}
+  > **Note:** DNS validation statuses for Dedicated Cloud Gateways are refreshed every 5 minutes.
+
+
+### Delete a custom domain {#delete-url}
+
+1. In {{site.konnect_short_name}}, open {% konnect_icon runtimes %} **Gateway Manager**, choose a control plane to open the **Overview** dashboard, then click **Custom Domains**.
+
+2. Click the hamburger menu on the right of the row you want to delete and click **Delete**.
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/dedicated-cloud-gateways/custom-dns.md:40]:
```suggestion
2. Click the action menu on the end of the row you want to delete and click **Delete**.
```
Hamburger menu is a bit of an english colloquialism/design jargon, should avoid it.

File: app/konnect/gateway-manager/serverless-gateways/custom-dns.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
+---
+title: Custom Domains
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/serverless-gateways/custom-dns.md:2]:
```suggestion
title: Custom Domains for Serverless Gateways
```

File: app/konnect/gateway-manager/serverless-gateways/custom-dns.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+{{site.konnect_short_name}} integrates domain name management and configuration with [Serverless](/konnect/gateway-manager/serverless-gateways/) gateways. 
+
+
+## Serverless Gateways
+### {{site.konnect_short_name}} configuration
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/serverless-gateways/custom-dns.md:9]:
```suggestion
## {{site.konnect_short_name}} configuration
```
I'm guessing that's a leftover from the original page.

File: app/konnect/gateway-manager/dedicated-cloud-gateways/custom-dns.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
+---
+title: Custom Domains
+---
+
+{{site.konnect_short_name}} integrates domain name management and configuration with [managed data planes](/konnect/gateway-manager/dedicated-cloud-gateways/).
+
+
+## Managed data planes
+### {{site.konnect_short_name}} configuration
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/dedicated-cloud-gateways/custom-dns.md:9]:
```suggestion
## {{site.konnect_short_name}} configuration
```

File: app/konnect/gateway-manager/dedicated-cloud-gateways/custom-dns.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+1. Open {% konnect_icon runtimes %} **Gateway Manager**, choose a control plane to open the **Overview** dashboard, then click **Connect**.
+    
+    The **Connect** menu will open and display the URL for the **Public Edge DNS**. Save this URL.
+
+2. Select **Custom Domains** from the side navigation, then **New Custom Domain**, and enter your domain name.
+
+    Save the values that appear under **CNAME** and **Content**. 
+
+
+### Domain registrar configuration
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/dedicated-cloud-gateways/custom-dns.md:20]:
```suggestion
## Domain registrar configuration
```

File: app/konnect/gateway-manager/dedicated-cloud-gateways/custom-dns.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+|---------------------------------|-------------|----------------|-------|------------------------|------------------------------------------------------|-----|
+| `_acme-challenge.example.com` | CNAME       | Simple         | No    | No                     | `_acme-challenge.9e454bcfec.acme.gateways.konghq.com`| 300 |
+| `example.com`             | CNAME       | Simple         | No    | No                     | `9e454bcfec.gateways.konghq.com`                     | 300 |
+
+
+  {:.note}
+  > **Note:** DNS validation statuses for Dedicated Cloud Gateways are refreshed every 5 minutes.
+
+
+### Delete a custom domain {#delete-url}
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/dedicated-cloud-gateways/custom-dns.md:36]:
```suggestion
## Delete a custom domain {#delete-url}
```

File: app/konnect/gateway-manager/serverless-gateways/custom-dns.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+| Host Name                       | Record Type | Routing Policy | Alias | Evaluate Target Health | Value                                                | TTL |
+|---------------------------------|-------------|----------------|-------|------------------------|------------------------------------------------------|-----|
+| `my.example.com`             | CNAME       | Simple         | No    | No                     | `9e454bcfec.kongcloud.dev`                     | 300 |
+
+
+  {:.note}
+  > **Note:** Once a Serverless Gateway custom DNS record has been validated, it will _not_ be refreshed or re-validated. Remove and re-add the custom domain in {{site.konnect_short_name}} to force a re-validation.
+
+
+### Delete a custom domain {#delete-url}
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/serverless-gateways/custom-dns.md:36]:
```suggestion
## Delete a custom domain {#delete-url}
```

File: app/konnect/gateway-manager/serverless-gateways/custom-dns.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+1. In {{site.konnect_short_name}}, open {% konnect_icon runtimes %} **Gateway Manager**, choose a control plane to open the **Overview** dashboard, then click **Custom Domains**.
+
+2. Click the hamburger menu on the right of the row you want to delete and click **Delete**.
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/serverless-gateways/custom-dns.md:40]:
```suggestion
2. Click the action menu on the end of the row you want to delete and click **Delete**.
```

**Comment by johnharris85** [comment]
Addressed all comments (I think), PTAL @lena-larionova :heart:

**Review by lena-larionova — APPROVED** [review]
LGTM!

**Pull request merged**